entrypoint.sh

```diff
@@ -63,6 +63,22 @@ else
 			echo "Package HASH check failed"
 			exit 1
 		fi
+
+		PATCHES_DIR=`find feeds -path "*/$PKG/patches"`
+		if [ -d "$PATCHES_DIR" ]; then
+			make \
+				BUILD_LOG="$BUILD_LOG" \
+				IGNORE_ERRORS="$IGNORE_ERRORS" \
+				QUILT_REFRESH_ARGS="--no-timestamps --no-index -p ab" \
```

Comment:
Isn't this default?

Reply:
it is listed on https://openwrt.org/docs/guide-developer/build-system/use-patches-with-buildsystem#prepare_quilt_configuration, I wasn't sure - can remove if not needed.

Reply:
```
~/d/openwrt > git grep no-timestamps
include/quilt.mk:               QUILT_DIFF_OPTS="-p" $(QUILT_CMD) refresh -p ab --no-index --no-timestamps; \
```

Reply:
Removed QUILT options.